lib/src/utils.rs

```diff
@@ -10,6 +10,25 @@ use ostree::glib;
 use ostree_ext::container::SignatureSource;
 use ostree_ext::ostree;
 
+/// Helpers intended for [`std::process::Command`].
+pub(crate) trait CommandRunExt {
+    fn run(&mut self) -> Result<()>;
+}
+
+impl CommandRunExt for Command {
```

Comment:
I see you are using Command::new and not CommandRunExt above. I am not sure I follow how this gets wired. I imagine that now everytime someone uses Command::new it will use this instead, but have no idea how that is happening.

Reply:
mmm I guess it's using `trait` and then the impl that wires this.

Reply:
Right, the trait adds a new method.